cmd/pulumi-test-language/README.md

```diff
@@ -25,11 +25,29 @@ For each test run and language, then:
   separate process and the assertions are checked, exercising program execution
   for the given language.
 * Generated code is snapshot tested to ensure that it doesn't change
-  unexpectedly.[^pulumi-accept]
+  unexpectedly.
 
-[^pulumi-accept]:
```

Comment:
I think this is kinda useful as a mention here still.  Maybe it can be linked as a footnote both here and below?  There's other kinds of snapshot tests that also generate data using `PULUMI_ACCEPT`.

Reply:
> There's other kinds of snapshot tests that also generate data using PULUMI_ACCEPT

This is on the page that's specific to language conformance tests though.